addons/web_editor/static/lib/odoo-editor/test/spec/format.test.js

```diff
@@ -684,6 +690,53 @@ describe('Format', () => {
                 contentAfter: '<p>ab[]cd</p>',
             });
         });
+        it('should change the font-size of a span that has already a font-size', async () => {
```

Comment:
That title is incorrect because the font-size of the span that already has a font-size is not changed.

Reply:
changed to
> should change the font-size for a character in an inline that has a font-size